docs/hub/Guides/mint-page-deploy.md

```diff
@@ -109,4 +111,12 @@ Fork the repository https://github.com/holaplex/hub-starter-mint and give it a u
 
 ### View your mint page
 
-Your mint page is now deployed and ready to use! The URL is available in Render, e.g. https://mint-page.onrender.com
\ No newline at end of file
+Your mint page is now deployed and ready to use! The URL is available in Render, e.g. https://mint-page.onrender.com
+
+### Troubleshooting
+
+If you have encountered any errors in creating wallets and customers while deploying your app, you may need to dump the database and restart:
+```
+npm run reset
+npm run generate
```

Comment:
Good to have notes on the reset to clear and recreate the db but npm run generate should be added as a must do step. Once environment is up go to shell and run `npm run db`.

Remove npm run generate that is only for development and just running npm reset will delete the database, re-create it and re-run migrations.

Reply:
thanks for the info!! making sure i understand - we've got `npm run db` in the instructions, under "Set up the database". and i updated the reset instructions to include only `npm run reset` and not `npm run generate`